### PR TITLE
colors updated for a11y and to match code sample styles to zeplin

### PR DESCRIPTION
**Pull request by BrittanyIRL**
# Overview 
This PR is to update the colors used for code samples and also to clean up theme colors and make sure contrasts are OK for a11y. 

## `<code>` styles 
- The prism styles that control the `<code>` blocks and markdown (see docs/anything) hadn't been updated yet. Here we have updated the colors to be inline with zeplin. I had to darken the foreground colors and lighten the background pink on the large code blocks to comply with a11y, they look like this: 

This branch: 
Screen Shot 2020-02-21 at 11 04 02 AM

vs in Zeplin: 
Screen Shot 2020-02-21 at 6 01 24 PM

(mostly it's that teal that's noticeably different). 

## Theme colors 
- While updating the prism styles I got kinda annoyed with how many different browns there were and that it was hard to tell them apart, combine that with running an axe check on color contrast and getting 900 errors from the various links that weren't contrasted enough anddd here we are! 
- Updated theme color ordering to go from lightest to darkest shade of that color
- updated the generic color descriptions to be their true (or as close as possible) color name
- removed colors that were redundant or just sligghttlly different (looking at you `darkGray`)
- adjusted darkness to comply with a11y contrast standards

There's only really one noticeable color change here, and it's on the home page.

This branch: 
Screen Shot 2020-02-21 at 11 04 16 AM

Zeplin: 
Screen Shot 2020-02-21 at 6 06 47 PM

### Palettes for reference
Screen Shot 2020-02-21 at 2 47 14 PM
Screen Shot 2020-02-21 at 2 48 22 PM
Screen Shot 2020-02-21 at 2 58 16 PM

## To Test 
- pull and run code locally, see that markdown now follows the styles set in zeplin for single lines and blocks

### CODE REVIEW

File: src/partials/home/get-started.js [hunk 1]
```diff
@@ -33,16 +33,16 @@ const StyledLandingSectionContent = styled(LandingSectionContent)`
...
         <SectionDescription>{description}</SectionDescription>
-        <LinkButton to={link.location} color={importedTheme.color.darkBrown}>
+        <LinkButton to={link.location} color={importedTheme.color.cabSav}>
           {link.text}
         </LinkButton>
       </StyledLandingSectionContent>
-      <Stripe bg={importedTheme.color.brown} />
-      <Stripe bg={importedTheme.color.otherBrown} />
+      <Stripe bg={importedTheme.color.rouge} />
+      <Stripe bg={importedTheme.color.crownOfThorns} />
```
Comment (boygirl) [inline src/partials/home/get-started.js]: 😂

Comment (BrittanyIRL) [reply src/partials/home/get-started.js]: I know right?!? I swear I didn't make it up.
Screen Shot 2020-02-21 at 5 54 08 PM

File: src/partials/sidebar/index.js [hunk 2]
```diff
@@ -143,11 +143,14 @@ const Sidebar = ({ className, content, location, onCloseClick }) => {
         filteredResults.filter(edge => {
           return edge.data && edge.data.type === sectionCategory.type;
         });
-      return (filteredByCategory[
-        [sectionCategory.category]
-      ] = filteredEdges.filter(edge =>
-        sectionCategory.category.includes(edge.data.category)
-      ));
+
+      return filteredEdges
```
Comment (BrittanyIRL) [inline src/partials/sidebar/index.js]: Fixes the `Failed exporting HTML for URL gallery/area-hover-styles (/Users/${user-repo-path}/victory-docs/src/pages/gallery-item-template): Cannot read property 'filter' of undefined` issue

**Comment by boygirl** [comment]
@BrittanyIRL would you mind pulling the build related commits into a separate PR?  I want to talk with Joe more about the new syntax highlighting.

**Comment by BrittanyIRL** [comment]
> @BrittanyIRL would you mind pulling the build related commits into a separate PR? I want to talk with Joe more about the new syntax highlighting.

yup, absolutely!

**Comment by BrittanyIRL** [comment]
> > @BrittanyIRL would you mind pulling the build related commits into a separate PR? I want to talk with Joe more about the new syntax highlighting.
> 
> yup, absolutely!

https://github.com/FormidableLabs/victory-docs/pull/661

**Pull request closed without merging**